appserver/payara-appserver-modules/microprofile/jwt-auth/src/main/java/fish/payara/microprofile/jwtauth/jwt/JwtTokenParser.java

```diff
@@ -72,6 +72,9 @@
     private final boolean enableNamespacedClaims;
     private final Optional<String> customNamespace;
 
+    private String rawToken;
+    private SignedJWT signedJWT;
```

Comment:
It's typically bad practice to have two class variables depending directly on each other. Is it possible to make the signed JWT a local variable without breaking the TCK or introducing another problem?

Reply:
Shouldn't be too much trouble, I'll cook something up.